### ISSUE TITLE
[Enhance] Tardis Dimension Properties

### ISSUE THREAD

**Issue by ghost** [DarkholmeTenk/TardisModIssues#477]
In the tardis the sky has the appearance of the overworld with a day and night option. I was wondering if it was possible to change what dimension the sky-box is using and/or take properties from certain dimensions when changed to that. For example in the nether water cannot be placed normally and has a red tint to the fog/sky with certain music normally played for that type of dimension and to be able to have those types of properties within the tardis dimension.

**Comment by AlienXtream** [comment]
already done. next to the spawn protection lever (on the engine) you can toggle between 2 skyboxes

**Comment by FoxPotato** [comment]
both options are possible, but the properties thing won't happen.
I've tried the skybox before though, might try and finish that.

**Comment by FoxPotato** [comment]
and alien, he meant have the TARDIS when in normal skybox take over the skybox of the dimension the exterior is in atm.. I think

**Comment by AlienXtream** [comment]
oh. that would be possible but FAR to tedious to warrant it. although an additional option to use botania's "Garden of glass" sky box would be cool. it has auroras :P

**Comment by AlienXtream** [comment]
actually, if there is a registry for custom skyboxes then could you use it to dynamically get and apply any custom skyboxes it finds?

**Comment by FoxPotato** [comment]
well it's actually not hard at all tbh
there's literally a function getskyrenderer. Which you have to override to get a custom renderer :P 
Like i said, I've tried it before. But i got a few problems, don't know which ones tho. My memory is the worst.

**Comment by AlienXtream** [comment]
Well if you DO have a stab at it than a disclaimer about the jittery motion during fixed time setting will be a must

**Comment by FoxPotato** [comment]
True, morph will still work. But the sky renderers might not like it.

**Comment by AlienXtream** [comment]
Given the minority of the render bug its a worth while sacrifice

**Comment by FoxPotato** [comment]
I'll probably make it a config option, when true display dimension sky. when false just display overworld.
